### PR TITLE
GUACAMOLE-901: Belgian French keymap

### PR DESCRIPTION
**Pull request by jukeks**

### CODE REVIEW

File: src/protocols/rdp/Makefile.am [hunk 1]
```diff
@@ -269,6 +269,7 @@ rdp_keymaps =                   \
     $(srcdir)/keymaps/en_us_qwerty.keymap \
     $(srcdir)/keymaps/es_es_qwerty.keymap \
     $(srcdir)/keymaps/fr_fr_azerty.keymap \
+    $(srcdir)/keymaps/fr_be_azerty.keymap \
```
Comment (necouchman) [inline src/protocols/rdp/Makefile.am]: Nitpick - this should probably be *above* the `fr_fr` keymap, as should the `fr_ch` one.

Comment (jukeks) [reply src/protocols/rdp/Makefile.am]: They are now alphabetically sorted.

**Comment by jukeks** [comment]
Elaborated the commit message a bit.

**Comment by necouchman** [comment]
@jukeks: Thank you, again, for the contribution.  Please note that the change will also need to be made on the guacamole-client side, as well, to add it as an option for use when configuring connections.

**Pull request merged**